### PR TITLE
Add video call link to section

### LINKED ISSUE #614: Add video call link to section
1) Add a field to section
2) Allow HR to add video call link when assigning/reassigning the section
3) Show video call link on section page
4) Add videocall link to calendar email

### PR DESCRIPTION
**Pull request by nyet-ty**
Resolve #614 

## PR includes

-   [x] Feature

### CODE REVIEW

File: src/components/CandidateNameSubtitle/CandidateNameSubtitle.tsx [hunk 1]
```diff
@@ -21,7 +22,7 @@ export function CandidateNameSubtitle({ id, name }: Props) {
                 lineHeight: '28px',
             }}
         >
-            Candidate: {typeof id !== 'number' ? name : <Link href={pageHrefs.candidate(id)}>{name}</Link>}
+            {tr('Candidate')}: {typeof id !== 'number' ? name : <Link href={pageHrefs.candidate(id)}>{name}</Link>}
```
Comment (awinogradov) [inline src/components/CandidateNameSubtitle/CandidateNameSubtitle.tsx:25]: just use nullable

File: src/components/CreateOrUpdateSectionForm/CreateOrUpdateSectionForm.tsx [hunk 2]
```diff
@@ -168,6 +171,7 @@ export const CreateOrUpdateSectionForm = ({
         setSchedulable(arg);
         setSearch('');
     };
+    const setVideoCallLink = (link: string) => setValue('videoCallLink', link);
```
Comment (awinogradov) [inline src/components/CreateOrUpdateSectionForm/CreateOrUpdateSectionForm.tsx:174]: I'm not sure I understand what is happening here

File: src/components/Section/Section.tsx [hunk 3]
```diff
@@ -92,6 +93,12 @@ export const Section = ({ section }: SectionProps): JSX.Element => {
                     <>
                         <CandidateNameSubtitle name={interview.candidate.name} id={interview.candidate.id} />
 
+                        {section.videoCallLink && (
```
Comment (awinogradov) [inline src/components/Section/Section.tsx:96]: nullable

**Review by awinogradov — CHANGES_REQUESTED** [review]

File: src/components/CandidateNameSubtitle/CandidateNameSubtitle.tsx (hunk 1 above)
Comment (nyet-ty) [reply src/components/CandidateNameSubtitle/CandidateNameSubtitle.tsx:25]: Why? There will be name of candidate or link to his interview

File: src/components/CreateOrUpdateSectionForm/CreateOrUpdateSectionForm.tsx (hunk 2 above)
Comment (nyet-ty) [reply src/components/CreateOrUpdateSectionForm/CreateOrUpdateSectionForm.tsx:174]: This function needed to control videoCallLink in [SectionScheduleCalendar](https://github.com/taskany-inc/hire/pull/622/files#diff-b5509b3f41dc4c72c051422583518d78c94d656aac493e495d7c2452836deb61), when section assigning in calendar

File: src/components/AddProblemToSection/AddProblemToSection.tsx [hunk 4]
```diff
@@ -14,6 +15,10 @@ const StyledModalHeader = styled(ModalHeader)`
     padding-bottom: ${gapM};
 `;
 
+const StyledButton = styled(Button)`
+    margin: ${gapM} 0;
```
Comment (awinogradov) [inline src/components/AddProblemToSection/AddProblemToSection.tsx:19]: It's not obvious why you need this style here. U have 3 buttons in a row, why not to add margin for container of them?

File: src/components/SectionFeedback/SectionFeedback.tsx [hunk 5]
```diff
@@ -46,6 +50,15 @@ const StyledButton = styled(Button)`
     margin: ${gapM} 0;
 `;
 
+const StyledButtonWrapper = styled.div`
+    display: flex;
+    gap: ${gapS};
+`;
+
+const StyledLinkWrap = styled.div`
+    margin: ${gapM} 0;
```
Comment (awinogradov) [inline src/components/SectionFeedback/SectionFeedback.tsx:59]: Same style with addProblemButton ?

**Review by awinogradov — APPROVED** [review]

**Review by 9teen90nine — APPROVED** [review]

**Pull request merged**